packages/ui/src/api/transport.ts

```diff
@@ -25,6 +27,14 @@ const transport = {
 
     const resp = await fetch(endpoint, init)
 
+    if (resp.headers.get('busmap-session-user') === 'inactive') {
+      try {
+        authn.postMessage('no-user-session')
```

Comment:
Will this work for api calls other than favorites which do not run in a worker so share the same browsing context? See if the message event fires for calls other than favorites …

Reply:
Confirmed this is working after creating a new channel instance in the `<Navigation />` component.